builtin/aws/ecr/mapper.go

```diff
@@ -0,0 +1,13 @@
+package ecr
+
+import (
+	"github.com/hashicorp/waypoint/builtin/docker"
+)
+
+// ECRImageMapper maps a ecr.Image to a docker.Image structure.
+func ECRImageMapper(src *Image) *docker.Image {
+	return &docker.Image{
```

Comment:
Does this need the `Location` field set? Or do we not know or assume its in a local registry?

Reply:
We assume it's remote, so probably need a Location, yeah.